src/app/containers/ConsentBanner/Banner/index.amp.test.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { shouldMatchSnapshot } from '../../../helpers/tests/testHelpers';
+import { ServiceContextProvider } from '../../../contexts/ServiceContext';
+import Banner from './index.amp';
+
+describe('Amp Consent Banner Container - Banner', () => {
+  shouldMatchSnapshot(
+    'should correctly render privacy banner',
+    <ServiceContextProvider service="news">
+      <Banner
+        type="privacy"
+        acceptAction={{ tap: [`cookieId.show, privacyId.hide`] }}
+        rejectAction={{ tap: [`cookieId.show, privacyId.hide`] }}
```

Comment:
These could be `"` rather than backticks couldn't they? Just checking why it was done like this?

Reply:
Ive made them single quotes to match :)